boards/esp32-heltec-lora32-v2/doc.txt

```diff
@@ -0,0 +1,210 @@
+
+Since the board is
+[open source hardware](https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series),
+a number of clones are available.
+
+\htmlonly<style>div.image img[src="https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/raw/master/InstallGuide/win-screenshots/WIFI_LoRa_32.png"]{width:500px;}</style>\endhtmlonly
+\image html "https://github.com/Heltec-Aaron-Lee/WiFi_Kit_series/raw/master/InstallGuide/win-screenshots/WIFI_LoRa_32.png" "Heltec WiFi Lora 32 V2"
+
+## <a name="hardware"> Hardware </a> &nbsp;&nbsp; [[TOC](#toc)]
+
+This section describes
+
+- the [MCU](#mcu),
+- the default [board configuration](#board_configuration),
+- [optional hardware configurations](#optional_hardware),
+- the [board pinout](#pinout).
+
+### <a name="mcu"> MCU </a> &nbsp;&nbsp; [[TOC](#toc)]
+
+Most features of the board are provided by the ESP32 SoC. The following
+table summarizes these features and gives an overview of which of these
+features are supported by RIOT. For detailed information about the ESP32,
+see section \ref esp32_mcu "MCU ESP32".
+
+<center>
+MCU         | ESP32     | Supported by RIOT
+------------|-----------|------------------
+Vendor      | Espressif | |
+Cores       | 1 or 2 x Tensilica Xtensa LX6 | 1 core
+FPU         | yes (ULP - Ultra low power co-processor) | no
+RAM         | 520 kByte SRAM <br> 16 kByte  RTC SRAM | yes
+ROM         | 448 kByte | yes
+Flash       | 512 kByte ... 16 MByte |  yes
+Frequency   | 240 MHz, 160 MHz, 80 MHz | yes
+Power Consumption | 68 mA @ 240 MHz <br> 44 mA @ 160 MHz <br> 31 mA @ 80 MHz <br> 5 uA in deep sleep mode | yes <br> yes <br> yes <br> no
+Timers      | 4 x 64 bit | yes
+ADCs        | 2 x SAR-ADC with up to 18 x 12 bit channels total | yes
+DACs        | 2 x DAC with 8 bit | yes
+GPIOs       | 34 (6 of them are only inputs) | yes
+I2Cs        | 2 | yes
+SPIs        | 4 | yes
+UARTs       | 3 | yes
+WiFi        | IEEE 802.11 b/g/n built in | yes
+Bluetooth   | v4.2 BR/EDR and BLE | no
+Ethernet    | MAC interface with dedicated DMA and IEEE 1588 support | yes
+CAN         | version 2.0 | yes
+IR          | up to 8 channels TX/RX | no
+Motor PWM   | 2 devices x 6 channels | yes
+LED PWM     | 16 channels | no
+Crypto      | Hardware acceleration of AES, SHA-2, RSA, ECC, RNG | no
+Vcc         | 2.5 - 3.6 V | |
+Documents   | [Datasheet](https://www.espressif.com/sites/default/files/documentation/esp32_datasheet_en.pdf) <br> [Technical Reference](https://www.espressif.com/sites/default/files/documentation/esp32_technical_reference_manual_en.pdf) | |
+</center>
+
+### <a name="board_configuration"> Board Configuration </a> &nbsp;&nbsp; [[TOC](#toc)]
+
+Heltec WiFi LoRa 32 V2 has the following on-board components:
+
+- SemTech SX1278 or SX1276 for LoRaWAN communication
+- OLED display
+
+There are two hardware versions of the board:
+
+- SemTech SX1276 for LoRaWAN communication in the 433 MHz band
+- SemTech SX1278 for LoRaWAN communication in the 868/915 MHz band
+
```

Comment:
Those two chips should be swapped as it was made in https://github.com/RIOT-OS/RIOT/pull/13389.

Reply:
Done